src/components/portfolio/WelcomeVideo.jsx

```diff
@@ -0,0 +1,71 @@
+import DeleteIcon from '../../assets/icons/delete.png';
+import FormUploadPortfolio from './FormUploadPortfolio';
+import Loader from '../common/Loader';
+import useMediaUploader from '../../util/PortfolioUploadMediaService';
+
+const WelcomeVideo = ({ profileVideoUrl }) => {
+  const {
+    showFileInput,
+    handleButtonClick,
+    error,
+    isLoading,
+    handleWelcomeVideoChange,
+    handleWelcomeVideoSubmit,
+    handleWelcomeVideoDelete,
+  } = useMediaUploader();
+
+  if (isLoading) return <Loader />;
+
+  return (
+    <div className="h-2/5 flex flex-col items-center justify-center bg-pureWhite p-4">
+      {error.message && <p className="text-red font-bold">{error.message}</p>}
+      {profileVideoUrl ? (
+        <>
+          <h3 className="mb-4 font-spartan font-semibold text-2xl">
+            Your welcome video
+          </h3>
+          <div className="relative lg:w-2/5">
+            <video aria-label="portfolio video" controls>
+              <source src={profileVideoUrl} type="video/mp4" />
+              <source src={profileVideoUrl} type="video/mpeg" />
+              <source src={profileVideoUrl} type="video/quicktime" />
+              <source src={profileVideoUrl} type="video/x-msvideo" />
+              Your browser does not support the video tag.
+            </video>
+            <button
+              aria-label="delete image"
```

Comment:
Also "delete video" you already have alt describing that nested inside this a delete image with "delete icon"